src/components/admin/VideoList.tsx

```diff
@@ -320,6 +321,9 @@ const VideoList = ({
               <Col span={3}>
                 <Checkbox value="F">Non CHECKED</Checkbox>
               </Col>
+              <Col span={3}>
+                <Checkbox value="G">Entity Plays</Checkbox>
```

Comment:
I think it's the opposite?
```suggestion
                <Checkbox value="G">Doesn't play</Checkbox>
```